Sources/AST/Statement/DoCatchStatement.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  DoCatchStatement.swift
+//  AST
+//
+//  Created by Ethan on 05/11/2018.
+//
+import Source
+import Lexer
+
+/// A do catch block.
+public struct DoCatchStatement: ASTNode {
+  // TODO(ethan): Once we have exceptions, change this error to have type `Exceptions`
```

Comment:
Rather than an inline TODO it would be more useful if you create a ticket. Please remove comment